docs/analysis_plan.md

```diff
@@ -0,0 +1,60 @@
+# Analysis plan
+
+This document describes the analysis plan to perform a retrospective evaluation of the impact of incorporating wastewater data on the forecast performance of a semi-mechanistic renewal-based model.
+Forecasts of COVID-19 hospital admissions will be generated weekly and evaluated
+against a the final observed hospital admissions data for all age groups.
+
+# Data
+
+This analysis will use publicly available data from RKI of COVID-19 hospital admissions and wastewater concentration data at the state-level in Germany during the 2024-2025 respiratory virus season.
+We will use GitHub's commit history of both repositories to generate snapshots of each data source available as of each forecast date.
+Weekly snapshots of both datasets will be created every Monday.
+
+## Hospital admissions data
+Hospital admissions data are reported by age group and state as 7-day rolling sums.
+We will fit and generate forecasts for all age groups, at the state-level.
+We will take the difference between successive days to compute daily hospital admissions incidence.
+
+
+### Addressing right-truncation
+The hospital admissions data provided are indexed by the data of the positive test -- which is sometimes well before patients end up hospitalised, leading to a right-truncation problem described in Wolffram et al.
+
+We will address this in one of two ways:
+1. By correcting the data for right-truncation prior to passing it to the model (pipeline approach, generally bad)
+2. Separately estimating a delay distribution for each region, and modifying the `wwinference` model to account for right-truncated data (still sort of pipelining approach, but slightly better)
+
+## Wastewater concentration data
+Data on wastewater concentrations of SARS-CoV-2 are available from individual treatment plants, indexed by sample collection date.
+Each wastewater treatment plant is contained within a state, and contains the metadata on the catchment area (population size) served by the wastewater treatment plant.
+We will assume all individuals served by the wastewater treatment plant reside within the state.
+We will jointly fit to all wastewater treatment plants within a state.
+
+# Models
+- `wwinference` (ww + hosp): This model will jointly fit to the daily hospital admissions at the state-level and the wastewater concentration data from the treatment plants within the state.
```

Comment:
are you going to add adjustment for flow or use flow normalised data? If available either of these options seem like a good idea

Reply:
Will look back at the data -- I am not sure they have flow data but if they do we can use it
They don't provide LOD which is a bit of a bummer, I haven't done a full EDA to see how many data points are non-detects